src/DateRange/DateRange.spec.tsx

```diff
@@ -108,7 +108,34 @@ describe("DateRange", () => {
       );
       expect(latestCall.error).toEqual("end date is before start date");
     });
-    it("returns the selected range with an error if the range is invalid", () => {
+    it("does not return an error if the same date is selected without showing times", () => {
+      const startingDate = new Date();
+      const endingDate = new Date(
+        startingDate.getFullYear(),
+        startingDate.getMonth(),
+        startingDate.getDate() + 6
+      );
```

Comment:
Shouldn't the ending date be the same as the starting date to match the assertion `if the same date is selected`?

Reply:
To better match our input use case, we wanted to start with a different date, then select the same end date. That said, I think we can change our variable names to better reflect this.

Reply:
and possibly change the test name to emphasize the fact that select = click instead of setup